src/module/NormalCourse.cpp

```diff
@@ -61,23 +61,25 @@ void NormalCourse::runNormalCourse()
        * 進む距離，目標スピード，スピードpid，ターンpid
        */
       = { {
-          { 600, baseSpeed, { 0.1, 0.0, 0.0 }, { 0.8, 0.01, 0.06 } },    // 第1区間
-          { 800, baseSpeed, { 0.1, 0.0, 0.0 }, { 0.8, 0.0, 0.07 } },     // 第2区間
-          { 1300, baseSpeed, { 0.1, 0.0, 0.0 }, { 0.8, 0.0, 0.04 } },    // 第3区間
-          { 760, 580, { 0.1, 0.0, 0.0 }, { 0.80, 0.01, 0.08 } },         // 第4区間
-          { 740, 520, { 0.1, 0.0, 0.0 }, { 0.80, 0.01, 0.07 } },         // 第5区間
-          { 470, 560, { 0.1, 0.0, 0.0 }, { 0.80, 0.01, 0.08 } },         // 第6区間
-          { 1280, baseSpeed, { 0.1, 0.0, 0.1 }, { 0.80, 0.01, 0.06 } },  // 第7区間
-          { 700, baseSpeed, { 1.8, 0.0, 0.1 }, { 0.8, 0.01, 0.06 } },    // 第8区間
-          { 950, 550, { 0.1, 0.0, 0.1 }, { 0.80, 0.01, 0.07 } },         // 第9区間
-          { 750, 550, { 0.1, 0.0, 0.1 }, { 1.0, 0.0, 0.08 } },           // 第10区間
-          { 1400, 620, { 0.1, 0.0, 0.1 }, { 1.0, 0.0, 0.08 } },          // 第11区間
+          { 600, baseSpeed, { 0.1, 0.0, 0.0 }, { 0.8, 0.01, 0.06 } },        // 第1区間
+          { 800, baseSpeed, { 0.1, 0.0, 0.0 }, { 0.8, 0.0, 0.07 } },         // 第2区間
+          { 1300, baseSpeed, { 0.1, 0.0, 0.0 }, { 0.8, 0.0, 0.04 } },        // 第3区間
+          { 760, baseSpeed - 7, { 0.1, 0.0, 0.0 }, { 0.80, 0.01, 0.08 } },   // 第4区間
+          { 740, baseSpeed - 14, { 0.1, 0.0, 0.0 }, { 0.80, 0.01, 0.07 } },  // 第5区間
+          { 470, baseSpeed - 10, { 0.1, 0.0, 0.0 }, { 0.80, 0.01, 0.08 } },  // 第6区間
+          { 1280, baseSpeed, { 0.1, 0.0, 0.1 }, { 0.80, 0.01, 0.06 } },      // 第7区間
+          { 700, baseSpeed, { 1.8, 0.0, 0.1 }, { 0.8, 0.01, 0.06 } },        // 第8区間
+          { 950, baseSpeed - 11, { 0.1, 0.0, 0.1 }, { 0.80, 0.01, 0.07 } },  // 第9区間
+          { 750, baseSpeed - 11, { 0.1, 0.0, 0.1 }, { 1.0, 0.0, 0.08 } },    // 第10区間
+          { 1400, baseSpeed - 2, { 0.1, 0.0, 0.1 }, { 1.0, 0.0, 0.08 } },    // 第11区間
 
           { 1500, baseSpeed / 2, { 0.1, 0.0, 0.1 }, { 0.8, 0.0, 0.08 } },  //進入ライン
           { 350, baseSpeed / 4, { 0.1, 0.0, 0.1 }, { 1.1, 0.0, 0.08 } },   //本番で要調整
 
       } };
-  const std::array<NormalCourseProperty, arraySize>& normalCourseProperty = isLeftCourse ? normalCoursePropertyL : normalCoursePropertyR;
+  // LコースならLコースのPID値を採用する。RコースならRコースのPID値を採用する。
+  const std::array<NormalCourseProperty, arraySize>& normalCourseProperty
```

Comment:
```suggestion
  const std::array<NormalCourseProperty, arraySize> normalCourseProperty
```

こっちのほうが良いって@Tamiyasが言ってた．